app/renderer/components/Inspector/Inspector.js

```diff
@@ -41,6 +41,18 @@ const downloadXML = (sourceXML) => {
   document.body.removeChild(element);
 };
 
+const downloadXML = (sourceXML) => {
+  const href = 'data:application/xml;charset=utf-8,' + encodeURIComponent(sourceXML);
+  const filename = 'source.xml';
+  downloadFile(href, filename);
+};
+
+const downloadScreenshot = (screenshot) => {
+  const href = `data:image/png;base64,${screenshot}`;
+  const filename = 'appium-inspector-screenshot.png';
```

Comment:
maybe it would make sense to add the current timestamp to the screenshot file name

Reply:
The file names for both screenshot and XML source now include the UTC timestamp. (Conversion to local time seemed like it would be quite messy)